packages/tools/e2e-tests/src/page-objects/tools-app/components/toolsHeader.component.ts

```diff
@@ -0,0 +1,33 @@
+import CardComponent from '../../react-ui/card.component';
+import NavHeaderComponent from '../../react-ui/navHeader.component';
+
+import type { Locator, Page } from '@playwright/test';
+
+export default class ToolsHeaderComponent extends NavHeaderComponent {
+  public networkCard: CardComponent;
+
```

Comment:
Can we use named exports, for easy importing and maintainability

(Applies to all the others as well. I stopped adding the comment from here)